kwasm-lemmas.md

```diff
@@ -41,6 +41,11 @@ Not however that K defines `X modInt N ==Int X modInt (-N)`.
        andBool X  <Int N
       [simplification]
 
+    rule #wrap(N, X) => X
+      requires 0 <=Int X
+       andBool X  <Int (1 <<Int N)
+      [simplification]
```

Comment:
Is this true anymore with the additional rules about `#wrap`? I think you may need an extra side-condition `N >=Int 0`